### DISCUSSION TITLE
愚人移山 | /story/game-over/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#613] [Announcements]
https://yuanfan.rbind.io/story/game-over/

**Comment by yihui** [comment]
我就想知道“垂带踏跺”这个名称是怎么找到的。以我一级警犬的搜索力找了很久都没能找出来。

**Reply by earfanfan** [reply]
可能因为我有阿木狗这个真狗作伴，所以我的狗头的狗运更好吧。

你先告诉我你是怎么搜索的？

**Reply by yihui** [reply]
我就是搜“台阶两侧的水泥斜坡坡叫神马名字”之类的。这问题已经有其他求知欲旺盛的青年们在“百度不知道”上问过，但答案跟没回答一样：https://zhidao.baidu.com/question/456967056247507045.html

**Reply by earfanfan** [reply]
原本我也是怎么都找不到，后来有次偶然在豆瓣上刷到一些古建筑解析之类的图，按“古建筑 石台阶两边的是什么”就找到接近的了。我也是才明白，其实我们在现代中国大城市看到的都是水泥做的台阶了，只是样式居然是来源于古建筑的。那么，国外的房子或者街道上的台阶两边是没有垂带石的么？

**Reply by yihui** [reply]
非常少见。台阶两边一般是扶手栏杆，要不就是啥都没有，纯台阶。我们这旮旯在户外好像连台阶都不太常见。
